test/e2e/kind.yaml

```diff
@@ -17,6 +17,10 @@ kubeadmConfigPatches:
 nodes:
 - role: control-plane
   image: kindest/node:${KIND_K8S_VERSION}@${KIND_K8S_DIGEST}
+- role: control-plane
```

Comment:
did you mean for this to be a worker as well? Can't have 2 control planes. I also don't think scaling control planes would add much to our testing.

Reply:
Yeah I will correct this in my next commit 👍🏼